Makefile

```diff
@@ -23,7 +23,8 @@ generate: root build
 		.root/src/$(PKG)/tests/omitempty.go
 
 	.root/bin/easyjson -all .root/src/$(PKG)/tests/data.go 
-	.root/bin/easyjson -snake_case .root/src/$(PKG)/tests/snake.go 
+	.root/bin/easyjson -all .root/src/$(PKG)/tests/required.go
```

Comment:
I think it's better to put the tests in 'data.go': other files exist due to the generator being run with a different command-line flags. It'll be faster to run the tests (reflection-based generator is somewhat slow) and there doesn't seem to be any real issues with a single file containing many tests in observable future.

BTW, thanks for writing tests! :)